sdk/src/wallet/types/mod.rs

```diff
@@ -42,8 +42,6 @@ pub struct OutputData {
     pub output: Output,
     /// The output ID proof
     pub output_id_proof: OutputIdProof,
-    /// If an output is spent
-    pub is_spent: bool,
```

Comment:
Maybe replace this with `pub fn is_spent(&self)`

Reply:
Good idea